src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -419,6 +419,14 @@ protected function getEnv($name)
         if (false !== $env = getenv($name)) {
             return $this->envCache[$name] = $env;
         }
+        if (false !== $i = strpos($name, '@')) {
```

Comment:
This is about if we should allow to override the lookup service by having `FOO@service` in our `$_ENV` var for example. I've no strong opinion on this, but it could make sense to do:

```php
if (false !== $i = strpos($name, '@')) {
    $value = $service->getEnv(substr($name, 0, $i));
} else {
   $default = new GetEnv();
   $value = $default->getEnv($name);
}
```